src/lexer.h

```diff
@@ -74,10 +75,10 @@ class Lexer
     void deprecation(const char *format, ...);
     void poundLine();
     unsigned decodeUTF();
-    void getDocComment(Token *t, unsigned lineComment);
+    void getDocComment(Token *t, unsigned lineComment, int newParagraph);
```

Comment:
I have no idea, been ages since I did much with C/C++: The ```lineComment``` var is a boolean, but typed as unsigned, so I figured there may have been a reason to avoid bool. And I went with signed because within the ```combineComments``` function, ```insertNewLine``` is signed.